www/js/util.js

```diff
@@ -2097,11 +2097,20 @@ function waitUntilDefined(obj, key, fn) {
     fn();
 }
 
-function chatDialog(div) {
+/*
+    God I hate supporting IE11
+    https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters
+    https://caniuse.com/#search=default%20function
+
+    This would be the ideal:
+    function chatDialog(div, zin = "auto") {
+*/
+function chatDialog(div, zin) {
+    if(!zin){ zin = 'auto'; }
     var parent = $("<div/>").addClass("profile-box")
         .css({
             padding: "10px",
-            "z-index": "auto",
+            "z-index": zin,
```

Comment:
I'm kinda curious why it was originally set to auto explicitly.  Having the default here at least avoids breaking anything by introducing the 9999 to just this one case, but at some point we may need to look at the other places where chatDialog is used to see if there are similar bugs.

Reply:
That was exactly my thinking, didn't know why it was like this and didn't want to break anything as a side effect.